assets/js/userCtrl.js

```diff
@@ -106,34 +107,7 @@ module.exports = function ($scope, $filter, $location, $timeout) {
   };
 
   $scope.formattedName = function (name) {
```

Comment:
Just bind this directly, no?

$scope.formattedName = flairsService.formattedName